product_docs/docs/pgd/4/rel_notes/index.mdx

```diff
@@ -47,3 +47,4 @@ The EDB Postgres Distributed documentation describes the latest version of EDB P
 
 
 [^*] We released a patch to HARP 2.2.1 to address a security vulnerability. If using 2.2.1 with an earlier version of EDB Postgres Distributed, we recommend that you upgrade to 2.2.2.
```

Comment:
maybe adjust this too? and recommend 2.2.3?

Reply:
Yes, this would have a separate line for 2.3.0. Creating a separate rel notes for that.